### PR TITLE
Checklist refactor using FormattedMessage

### PR DESCRIPTION
**Pull request by wackerow**
- Updated approximate Geth size
- Added recommendation to not wait on eth1 chain syncing
- Refactored Checklist to use `<FormattedMessage />` component and `formatMessage()`
- Provided descriptions whenever "values" attribute is used to assist translators. 

@samajammin Lemme know if this looks correct to prepare for crowdin

### CODE REVIEW

File: src/pages/Checklist/index.tsx [hunk 1]
```diff
@@ -246,80 +286,137 @@ export const Checklist = () => {
...
               </Text>
             </li>
             <li className="py5">
               <Text>
-                <strong>Note:</strong> by default, VMs may disable NTP so you
-                may need to find a work-around for your environment.
+                <FormattedMessage
+                  defaultMessage="{note}: by default, VMs may disable NTP so you may need to find a work-around for your environment."
+                  values={{
+                    note: <strong>Note</strong>,
```
Comment (samajammin) [inline src/pages/Checklist/index.tsx:370]: This is tricky & something we may want to choose to just remove the bold on "Note".

If we want to keep the bold, we'd still want to translate "Note", so the value should be:
```
note: <strong>formatMessage({defaultMessage: "Note"})</strong>
```
But that would take "Note" out of context. Translators may thinks it's the noun vs. the verb.

Comment (samajammin) [reply src/pages/Checklist/index.tsx:370]: We could use a `description` to explain it's the verb but the lack of tense general obscurity might make it too confusing to translators for this to be worth it.

Comment (wackerow) [reply src/pages/Checklist/index.tsx:370]: Yeah, the description was how I was attempting to go about this. Removing the bold would likely be the best option. These aren't really warnings, and don't demand a high level of attention. 
And good call on using `formatMessage()` to translate within the `values` attribute... wasn't thinking, I can add another commit to fix any of these.

File: src/pages/Checklist/index.tsx [hunk 2]
```diff
@@ -246,80 +280,127 @@ export const Checklist = () => {
...
-                <Code>Universal time</Code> are all correct.
+                <FormattedMessage
+                  defaultMessage="Check if {code1}, {code2}, and {code3} are all correct."
+                  values={{
+                    code1: <Code>Local time</Code>,
+                    code2: <Code>Time zone</Code>,
+                    code3: <Code>Universal time</Code>,
+                  }}
+                  description="{code} values are terminal outputs to look for to confirm system time"
+                />
```
Comment (wackerow) [inline src/pages/Checklist/index.tsx:326]: @samajammin honestly not sure if these should be translated or not... I would think a user's system settings would produce these in their native language, but not sure.

File: src/pages/Checklist/index.tsx (hunk 1 above)
Comment (samajammin) [reply src/pages/Checklist/index.tsx]: I agree, in general I suspect the value of bolding words won't be worth the effort/potential confusion. I think this feature will be used best with variables that we do NOT want translated (such as code snippets / CLI commands).

File: src/pages/Checklist/index.tsx (hunk 2 above)
Comment (samajammin) [reply src/pages/Checklist/index.tsx:326]: Good question. @CarlBeek do you know anyone who might know this? Does `timedatectl ` support multiple languages? A quick Google didn't yield any aswers.

Comment (samajammin) [reply src/pages/Checklist/index.tsx:326]: If in doubt I suggest we keep it in English.

Comment (wackerow) [reply src/pages/Checklist/index.tsx:326]: Okay... this is set up to not be translated at the moment, I will leave it pending any new information.

**Review by samajammin — APPROVED** [review]

File: src/pages/Checklist/index.tsx (hunk 2 above)
Comment (samajammin) [reply src/pages/Checklist/index.tsx:326]: Merging this for now. @CarlBeek please let us know if you're aware of any `timedatectl` language support.

**Pull request merged**